### ISSUE TITLE
Agent Merge Request - Collapse J. V. Moody to James V. Moody

### ISSUE THREAD

**Issue by mvzhuang** [ArctosDB/arctos#5202]
Use this template when you discover two agents that you believe represent the same person or organization.

**Agent that should remain after the merge**
James V. Moody

**Agent that will no longer remain after the merge**

J. V. Moody: https://arctos.database.museum/agents.cfm?agent_id=21319882

and sort of Moody Fronoke Hall ??? https://arctos.database.museum/agents.cfm?agent_id=21319816 

**Reason for merge**
Please provide the reason you believe that these two agents represent the same person or organization.

J.V. Moody is co-collector of Oscar Francke and J. B. Hall in UTEP data. James V. Moody was a Master's Thesis student under Francke. - Must be same person. 
Agent Moody Fronoke Hall should be James V. Moody and J. B. Hall (don't know first name). Fronoke is probably typo for Oscar F. Francke.

Please add this issue to the Agents Project

@lin-fred 
@ArctosDB/agents-committee

**Comment by ewommack** [comment]
I almost wonder if you need to split Agent Moody Fronoke Hall into James V. Moody, J. B. Hall, and Oscar F. Francke.
And then merge them into the other people?

And are you thinking that all three people should be attributed to everything attributed to Agent Moody Fronoke Hall? If so, maybe add James V. Moody, J. B. Hall, and Oscar F. Francke to all of those records. And then after that delete the weird mish-mashed agent?

**Comment by mvzhuang** [comment]
Thanks Beth. Your suggestion was way less complicated than what i was thinking

**Comment by lin-fred** [comment]
Good catch on this one! I will add James V. Moody and Oscar F. Francke as collectors for NMMNH:Ento:3895, but otherwise I think Moody Fronoke Hall should just be deleted, so merged into unknown and I will go ahead and do that.

And I am ok with J. V. Moody merging into James V. Moody,

**Comment by lin-fred** [comment]
J. V. Moody is now set to merge into James V. Moody

and Moody Fronoke Hall is now set to merge into unknown
